website/docs/r/vpc_peering_connection.html.markdown

```diff
@@ -22,6 +22,10 @@ management of the VPC Peering Connection and allows options to be set correctly
 VPC Peering Connections use the `aws_vpc_peering_connection` resource to manage the requester's side of the
 connection and use the `aws_vpc_peering_connection_accepter` resource to manage the accepter's side of the connection.
 
+-> **Note:** Creating multiple `aws_vpc_peering_connection` resources with the same `peer_vpc_id` and `vpc_id` will not produce an error.
+Instead, AWS will return the connection `id` that already exists, resulting in multiple `aws_vpc_peering_connection` resources with
+the same `id`.
```

Comment:
I _think_ in order for the `~>` syntax to work, you need to not have line breaks

```suggestion
-> **Note:** Creating multiple `aws_vpc_peering_connection` resources with the same `peer_vpc_id` and `vpc_id` will not produce an error. Instead, AWS will return the connection `id` that already exists, resulting in multiple `aws_vpc_peering_connection` resources with the same `id`.
```